site/content/en/blog/releases/1.8.0.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "1.8.0 - SDK Improvements, Including a New Unreal SDK!"
+linkTitle: "1.8.0"
+date: "2020-08-18"
+---
+This is the 1.8.0 release of Agones.
+
+Lots of changes to SDKs in this release, including a replacement Unreal SDK.
+
+The previous Unreal SDK will continue to work, but we recommend users upgrade to this latest version, as it has far
+more functionality.
+
+We also see more SDKs supporting the alpha Player Tracking functionality as well.
+
+Check the <a href="https://github.com/googleforgames/agones/tree/release-1.8.0" data-proofer-ignore>README</a> for details on features, installation and usage.
+
+**Breaking changes:**
+
+- \[Discussion\] Assimilate netspeakgames/UnrealAgonesSDK [\#1683](https://github.com/googleforgames/agones/issues/1683)
+- Upgrade to Kubernetes 1.15 [\#1478](https://github.com/googleforgames/agones/issues/1478)
```

Comment:
I think we did this in an earlier release, but didn't close out the github issue in a timely manner.

Reply:
https://agones.dev/site/blog/2020/05/26/1.6.0-player-tracking-kubernetes-1.15-node.js-sdk-updates/

Reply:
Agreed, but this is where the issue landed in the changelog.

Do you think I remove it from here? I can't remove it from the changelog, since it's auto-generated. (it's also in the RC release notes as well)

Reply:
I think it'll move if you change the milestone on the issue and re-run the generator for release notes. 

(and I don't think it should have been on the RC either, but I didn't review that PR 😄 )

Reply:
Audit trail from chat:

Agreed to drop this back to 1.16 and regen the changelog and site.

Reply:
and done!